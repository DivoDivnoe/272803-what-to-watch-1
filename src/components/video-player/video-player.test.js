import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

const mock = {
  image: `https://vokrug.tv/pic/news/d/d/8/4/dd84fb57fee799d831753daa2e0eb07e.jpg`,
  preview: ``,
  handleLoaded: jest.fn(),
  isPlaying: false,
};

describe(`VideoPlayer component`, () => {
  it(`renders correctly`, () => {
    const {image, preview, handleLoaded, isPlaying} = mock;

    const tree = renderer.create(
        <VideoPlayer
          image={image}
          preview={preview}
          handleLoaded={handleLoaded}
          isPlaying={isPlaying}
        />,
        {createNodeMock: (el) => {
          return el;
        }}).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
