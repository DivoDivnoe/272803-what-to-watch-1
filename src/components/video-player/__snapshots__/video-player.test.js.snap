// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer component renders correctly 1`] = `
<video
  className="f"
  height="150"
  poster=""
  width="300"
>
  Your browser does not support the video tag.
</video>
`;
